test.sh

```diff
@@ -25,11 +25,17 @@ else
 	PYTHON="docker run --rm -it -v $(pwd):/minikube -w /minikube python python"
 fi
 
-# Run "go test" on packages that have test files.
+
+# Run "go test" on packages that have test files.  Also create coverage profile
 echo "Running go tests..."
 cd ${GOPATH}/src/${REPO_PATH}
-TESTS=$(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...)
-go test -v ${TESTS}
+rm -f out/coverage.txt
+echo "mode: count" > out/coverage.txt
+for pkg in $(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...); do
```

Comment:
Sure, I can definitely add comments to this.  The reason that I added the coverage generation to the tests is that I believe that generating test coverage and testing are coupled.  The only way I know to generate coverage profiles is through go test.  I believe having a separate `make coverage` would require rerunning all of the tests in travis.